### PR TITLE
Interim deploy replacement scripts [DO NOT MERGE!]

### PR DESCRIPTION
**Pull request by tjcloa**

### CODE REVIEW

File: scripts/contractInteraction/contract_interaction.py [hunk 1]
```diff
@@ -35,8 +35,8 @@ def main():
     
     #executeOnMultisig(1071)
    
-    #confirmWithMS(1064)
-    #checkTx(1077)
+    #confirmWithMS(1139)
```
Comment (cwsnt) [inline scripts/contractInteraction/contract_interaction.py:38]: Why is this activated?

File: scripts/contractInteraction/contract_interaction.py [hunk 2]
```diff
@@ -35,8 +35,8 @@ def main():
...
+    #checkTx(1139)
```
Comment (cwsnt) [inline scripts/contractInteraction/contract_interaction.py:39]: Why is this activated?

File: scripts/contractInteraction/contract_interaction.py (hunk 1 above)
Comment (tjcloa) [reply scripts/contractInteraction/contract_interaction.py:38]: just signing leftovers
not material while commented out as the PR is not gonna be merged

File: scripts/contractInteraction/contract_interaction.py (hunk 2 above)
Comment (tjcloa) [reply scripts/contractInteraction/contract_interaction.py:39]: just signing leftovers
not material while commented out as the PR is not gonna be merged

**Pull request closed without merging**